10-1/10-1/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _10_1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int n = 0;
+            if (!TryInputNumber("Введите число n", out n))
+            {
+                Console.ReadKey();
+                return;
+            }
+
+            if (n < 0)
+            {
+                Console.WriteLine("Условие n - натуральное число не выполняется");
+                Console.ReadKey();
+                return;
+            }
+            0
+            double sum = 0;
+
+            for (var i = 1; i <= n; i++)
+            {
+                sum += (i/(i+1.0));
+                Console.WriteLine(i/(i+1.0));
+                Console.WriteLine(sum);
+            }
```

Comment:
Промежуточные результаты печатать не нужно. А если n = 1 000 000?
```suggestion
            for (var i = 1; i <= n; i++)
                sum += (i/(i+1.0));
```